tests/test_create_dataset_manifest.py

```diff
@@ -300,6 +300,47 @@ def test_object_detection_bbox_format_LTRB(self):
             self.assertEqual(dataset_manifest.images[0].labels, [[0, 10, 10, 100, 100]])
             self.assertEqual(dataset_manifest.images[1].labels, [[0, 100, 100, 200, 200], [1, 20, 20, 200, 200]])
 
+    def test_image_caption(self):
```

Comment:
can we add a test for multitask case where one task is caption and one task is classification? Thanks